spikeextractors/extraction_tools.py

```diff
@@ -842,4 +891,41 @@ def cast_start_end_frame(start_frame, end_frame):
         end_frame = end_frame
     else:
         raise ValueError("end_frame must be an int, float (not infinity), or None")
-    return start_frame, end_frame
\ No newline at end of file
+    return start_frame, end_frame
+
+
+def divide_recording_into_time_chunks(num_frames, chunk_size, padding_size):
+    chunks = []
+    ii = 0
+    while ii < num_frames:
+        ii2 = int(min(ii + chunk_size, num_frames))
+        chunks.append(dict(
+            istart=ii,
+            iend=ii2,
+            istart_with_padding=int(max(0, ii - padding_size)),
+            iend_with_padding=int(min(num_frames, ii2 + padding_size))
+        ))
+        ii = ii2
+    return chunks
+
+
+def _write_dat_one_chunk(i, rec_arg, chunks, rec_memmap, dtype, time_axis, verbose):
+    chunk = chunks[i]
```

Comment:
Do we really need the verbsity ?

Reply:
It's nice if someone wants to check the progress